apps/web/pages/event-types/index.tsx

```diff
@@ -930,6 +931,7 @@ const EventTypesPage = () => {
   const orgBranding = useOrgBranding();
   const routerQuery = useRouterQuery();
   const filters = getTeamsFiltersFromQuery(routerQuery);
+  useTheme(user?.appTheme);
```

Comment:
Do we only want to do this in `event-types/index` this should probably be handled in `Shell:356` - This is where we also handle settings brandColours for a user